doc/install/dependencies.rst

```diff
@@ -82,8 +82,7 @@ is required.
 ::
```

Comment:
Can we remove the note just above this saying that `--enable-dagmc` is required?